src/Microsoft.Extensions.SecretManager.Tools/Internal/ProjectIdResolver.cs

```diff
@@ -35,32 +36,37 @@ public string Resolve(string project, string configuration = Constants.DefaultCo
             var outputFile = Path.GetTempFileName();
             _tempFiles.Add(outputFile);
 
-            var commandOutput = new List<string>();
-            var commandResult = Command.CreateDotNet("msbuild",
-                new[] {
-                    targetFile,
-                    "/nologo",
-                    "/t:_FindUserSecretsProperty",
-                    $"/p:Project={projectFile}",
-                    $"/p:OutputFile={outputFile}",
-                    $"/p:Configuration={configuration}"
-                })
-                .CaptureStdErr()
-                .CaptureStdOut()
-                .OnErrorLine(l => commandOutput.Add(l))
-                .OnOutputLine(l => commandOutput.Add(l))
-                .Execute();
-
-            if (commandResult.ExitCode != 0)
+            var args = new[]
             {
-                _logger.LogDebug(string.Join(Environment.NewLine, commandOutput));
-                throw new GracefulException(Resources.FormatError_ProjectFailedToLoad(projectFile));
+                "msbuild",
+                targetFile,
+                "/nologo",
+                "/t:_FindUserSecretsProperty",
+                $"/p:Project={projectFile}",
```

Comment:
Is the correct way to escape arguments like this: `foo.exe "/p:foo bar" "/p:baz qux"`

Or like this? `foo.exe /p:"foo bar" /p:"baz qux"`

Because the former looks kind of wrong to me - though I would imagine it depends on how any particular command line tool actually works.

Does this actually work with MSBuild?